backend/videoService/videoServiceInterface.ts

```diff
@@ -7,4 +7,6 @@ export interface VideoService {
   startArchive(roomName: string, sessionId: string): Promise<Archive | SingleArchiveResponse>;
   stopArchive(archiveId: string): Promise<string>;
   listArchives(sessionId: string): Promise<Archive[] | SingleArchiveResponse[] | undefined>;
+  enableCaptions(sessionId: string): Promise<EnableCaptionResponse | string | undefined>;
```

Comment:
When do we return `string` or `undefined`?

Reply:
great catch, we only return `EnableCaptionResponse`. The other two came from before I added the said `EnableCaptionResponse`. I'll remove it. thanks! 💪